FRW/Modules/Register/ViewModel/UsernameViewModel.swift

```diff
@@ -99,12 +99,16 @@ class UsernameViewModel: ViewModel {
         Task {
             do {
                 let txid = try await UserManager.shared.register(currentText, mnemonic: nil)
-                let viewModel = CreateProfileWaitingViewModel(txId: txid ?? "") { _ in
+                let viewModel = CreateProfileWaitingViewModel(txId: txid ?? "") { _, createBackup in
 
                     DispatchQueue.main.async {
                         self.changeBackupTypeIfNeeded()
                         self.state.isRegisting = false
                         Router.popToRoot()
+                        
```

Comment:
Might need to add a delay?

Reply:
I thought about that, but it worked without any issue in my testing. Maybe I can test on a couple more devices and iOS versions, just to be sure